lib/publiccloud/provider.pm

```diff
@@ -264,7 +264,7 @@ sub terraform_apply {
     my $create_extra_disk = 'false';
     my $extra_disk_size   = 0;
 
-    $args{count} //= '1';
+    my $instance_count       = get_var('PUBLIC_CLOUD_INSTANCE_COUNT');
```

Comment:
```suggestion
    $args{count} //= get_var('PUBLIC_CLOUD_INSTANCE_COUNT','1');
```
and :
1. you don't need change below 
2. you not forcing anyone define this variable in case they ok with '1'

Reply:
ah sure!  perfecto! thx @anton

Reply:
np ,  but keep in mind that even without your patch there already ability to bypass this `count` via args variable from places where `terraform_apply` is called . Without seeing whole picture : 
how and where you will call create_instance function which is actually calling `terraform_apply` it is hard to say if your patch actually make sense or not . So I am suggesting to keep it for now and ammend with another which would actually add "some fabrics/rdma/hpc tests" and then decide how better to proceed here

Reply:
thx @anton. Only now I realized I can simply for create_instance(count => 2);
my bad and actually you helped me realizing how this works!

Reply:
Yes, exactly. The way to handle how many VMs you want to create is form the test itself. But please be aware that 

    create_instance(count => 2)

is not the same as doing twice

    create_instance()
    create_instance()

since both VMs would not end up in the same resource group or subnet in case you need connectivity. And it even would probably fail because we are calling twice `terraform apply` using the same .tf...  maybe some code improvement for the future...